internal/clients/etherscan/pubkey-finder_test.go

```diff
@@ -0,0 +1,109 @@
+package etherscan
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestGetFromResultHash(t *testing.T) {
+	assert := assert.New(t)
+
+	mockTxResult := []txResult{
+		{From: "address1", Hash: "aaa111"},
+		{From: "address2", Hash: "bbb222"},
+		{From: "address3", Hash: "ccc333"},
+	}
+
+	type args struct {
+		address string
+		txList  *txList
+	}
+
+	testCases := []struct {
+		name    string
+		args    args
+		want    common.Hash
+		wantErr bool
+	}{
+		{
+			"empty-transaction-list",
```

Comment:
another convention used is to prefex tests that expect an error e.g. `wantErr = true` with `err-` can you rename this to `"err-empty-transaction-list"`